docs/ChangeLog.md

```diff
@@ -1,3 +1,43 @@
+# 0.46.0
+Infrastructure:
+* Kotlinx Datetime JVM 0.5.0
+* Joda Time 2.12.5
+* Kotlinx Serialization Json 1.6.2
+* log4j2 2.22.0
+* slf4j 2.0.9
+* MariaDB (V3) driver 3.3.1
+* PostgreSQL driver 42.7.1
+* h2-database (V2) driver 2.2.224
+* SQLite driver 3.44.1.0
+* Spring Framework 6.1.2
+* Spring Boot 3.2.0
+* Spring Security Crypto 5.8.8
+
+Breaking changes:
+* chore!: EXPOSED-239 Set `preserveKeywordCasing` flag to true by default by @bog-walk in https://github.com/JetBrains/Exposed/pull/1948
+* More details at [Breaking changes](BREAKING_CHANGES.md#0460)
+
+Features:
+* feat: EXPOSED-65 Design query DSL consistent with SQL language by @bog-walk in https://github.com/JetBrains/Exposed/pull/1916
+* More details in the [Migration guide](MIGRATION_GUIDE.md#migrating-from-0450-to-0460)
+
+Bug fixes:
+* perf: EXPOSED-204 Performance problem with getConnection() by @bog-walk in https://github.com/JetBrains/Exposed/pull/1943
```

Comment:
If there's a better place for performance issues, or it needs its own heading, please let me know.